README.md

```diff
@@ -20,8 +20,9 @@ cd omnilrs_ros2_demo
 cd omnilrs_ros2_demo
 ./docker/run_container.sh
 
-# then build package
+# then build package & source
 colcon build --symlink-install
+source install/setup.bash
 ```
 
 
```

Comment:
suggestion: take this opportunity to add to the readme that you expect omnilrs to be running already

for example: link to omnilrs repo. and say you expect the environment to run with 
```python.sh run.py environment=lunaryard_20m``` ?